asset-transfer-basic/chaincode-go/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/hyperledger/fabric-samples/chaincode/fabcar/go
+
+go 1.13
```

Comment:
master Fabric is on go 1.14, wouldn't it be better to be on the same version here?

Reply:
done.